### DISCUSSION TITLE
Using serial interface on Raspberry PI 5

### DISCUSSION THREAD

**Discussion by Tux-Master** [Pi4J/pi4j#356] [General]
Is there a way to use the Serial Interface on the Raspberry PI 5 via PI4J?
 When checking the providers I find
gpiod - supports PI 5 but not serial
linuxfs - supports PI 5 but not serial
pigpio - does not support PI5 but supports serial
So there is no option for PI5 and serial? 
Are there any plans on when to support that?

**Comment by FDelporte** [comment]
We are not sure if Pi4J needs to provide a serial interface after all. The jSerialComm library is a very good alternative.
See more in https://github.com/Pi4J/pi4j-v2/discussions/308

**Reply by Tux-Master** [reply]
Not exactly what I wanted to hear :-)  coming from an rather old PI using PI4J V1.3. This means that i have to do more adaptions than I thought. But at least a hint why it is not working .. Thank you for the  fast response.
